### PR TITLE
python-eve: init at 0.6.1

### PR DESCRIPTION
**Pull request by Denommus**

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -7835,6 +7895,24 @@ in modules // {
...
+  flask-pymongo = buildPythonPackage rec {
+    name = "Flask-PyMongo-${version}";
+    version = "0.3.1";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/F/Flask-PyMongo/${name}.tar.gz";
+      sha256 = "0305qngvjrjyyabf8gxqgqvd9ffh00gr5yfrjf4nncr2my9svbyd";
+    };
+
+    pythonPath = with self; [ flask pymongo_2_9_1 ];
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: why `pythonPath` and not `propagatedBuildInputs`?

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -7835,6 +7895,24 @@ in modules // {
...
+
+    meta = {
+      homepage = "http://flask-pymongo.readthedocs.org/";
+      description = "PyMongo support for Flask applications";
+      license = licenses.bsd;
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: this license does not exist, `bsd2`, `bsd3` or `bsdOriginal` if I am not mistaken

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -4522,6 +4537,35 @@ in modules // {
...
+      markupsafe
+      pymongo
+      simplejson
+      werkzeug
+    ];
+
+    meta = {
+      homepage = "http://python-eve.org/";
+      description = "open source Python REST API framework designed for human beings";
+      license = licenses.bsd;
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: same

File: pkgs/top-level/python-packages.nix [hunk 4]
```diff
@@ -15920,6 +15998,24 @@ in modules // {
...
+  pymongo_2_9_1 = buildPythonPackage rec {
+    name = "pymongo-2.9.1";
+    version = "2.9.1";
+
+    src = pkgs.fetchurl {
+      url = "http://pypi.python.org/packages/source/p/pymongo/${name}.tar.gz";
+      sha256 = "1nrr1fxyrlxd69bgxl7bvaj2j4z7v3zaciij5sbhxg0vqiz6ny50";
+    };
+
+    doCheck = false;
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: could you explain here or in the commit why tests are disabled?

**Comment by FRidh** [comment]
I suggest you split the commit up in several, with one for each package. This makes it a lot easier for those in the future searching for these packages in say the git log.

**Comment by Denommus** [comment]
@FRidh I'll fix these mistakes and some more I just detected and reopen this PR later.

**Pull request closed without merging**